src/internal.c

```diff
@@ -13881,6 +13930,41 @@ PRAGMA_GCC_DIAG_POP
         alreadySigner = AlreadySigner(SSL_CM(ssl), subjectHash);
     }
 
+#ifdef WOLFSSL_DUAL_ALG_CERTS
+    if ((ret == 0) && (args->dCert->sapkiDer != NULL)) {
+#ifdef WOLFSSL_SMALL_STACK
+        byte *der = XMALLOC(MAX_CERT_VERIFY_SZ, ssl->heap, DYNAMIC_TYPE_DCERT);
+        if (der == NULL) {
+            ret = MEMORY_E;
+        }
+#else
+        byte der[MAX_CERT_VERIFY_SZ];
```

Comment:
Doesn't look good to me. Don't want to see variable declarations after code even if it is separated with #ifdefs.
Swap this around so that we have #ifndef WOLFSSL_SMALL_STACK